packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -549,6 +551,21 @@ export const PersonalDetailsForm = ({
         if (!form_initial_values.employment_status) form_initial_values.employment_status = '';
     }
 
+    const is_poa_verified = authentication_status?.document_status === 'verified';
+    const is_poi_verified = authentication_status?.identity_status === 'verified';
+
+    const is_account_verified = is_poa_verified && is_poi_verified;
+
+    //Generate Redirection Link to user based on verifiction status
+    const getRedirectionLink = () => {
+        if (!is_account_verified || !is_poi_verified) {
+            return '/account/proof-of-identity';
+        } else if (!is_poa_verified) {
+            return '/account/proof-of-address';
+        }
```

Comment:
@suisin-deriv here when poa is failed, `is_account_verified` is false and it will redirect to '/account/proof-of-identity' . isnt?   it shold be directed to poa page. right?

Reply:
@amina-deriv True will change condition 🙏